### DISCUSSION TITLE
Looking for ideas for creating a `Layout` with 50% of it's area as a popup, and the other as a selectable menu

### DISCUSSION THREAD

**Discussion by alextricity25** [MunifTanjim/nui.nvim#305] [Q&A]
Hey folks 👋 

I'm looking to create a UI that will show a selectable `Menu` component on the right side of a floating window, and on the left it will show a `Popup`. 

I attempted this look by using the `Layout` component, but maybe my understanding of how that works is incorrect. This is what I tried:
```
    bottom_popup = Popup({
        enter = true,
        border = bottom_popup_border
    })

    layout = Layout(
        {
            position = "50%",
            size = {
                width = "90%",
                height = "90%",
            },
        },
        Layout.Box({
            Layout.Box(bottom_popup, { size = "50%" }),
            -- tried adding menu here, but I get an error. Is this possible?
            Layout.Box(menu, { size = "50%" }),
        }, { dir = "row" })
    )
```
If this is not achievable using the `Layout` component, then are there any other suggestions?

**Comment by MunifTanjim** [comment]
This should be fixed by https://github.com/MunifTanjim/nui.nvim/pull/318
